### ISSUE TITLE
[Help] How to filter/change which quests I see?

### ISSUE THREAD

**Issue by Zeffuro** [Skamer/Syling-Tracker#115]
Hey there, I was just replacing Kaliel's Tracker with this since Kaliel's Tracker is entirely broken now and I noticed I'm not seeing all quests. 
I don't know if that's because it's autofiltering something, is me not using the config correctly or some other issue?

https://github.com/user-attachments/assets/21dfb1ba-739d-4329-8ff1-04b08c309185

I've added a video, I see no change in the amount of quests I see with Questie and other quest related addons disabled either.

My bugsack only contains errors relating to RaiderIO and an ElvUI error which I think is most likely irrelevant but here it is anyway in case I'm wrong: 
```
6x [ADDON_ACTION_BLOCKED] AddOn 'ElvUI_Libraries' tried to call the protected function 'CheckInteractDistance()'.
[string "@!BugGrabber/BugGrabber.lua"]:485: in function <!BugGrabber/BugGrabber.lua:485>
[string "=[C]"]: ?
[string "@ElvUI_Libraries/Core/LibRangeCheck-3.0-26/LibRangeCheck-3.0.lua"]:634: in function `interactCheck'
[string "@ElvUI_Libraries/Core/LibRangeCheck-3.0-26/LibRangeCheck-3.0.lua"]:663: in function `checker'
[string "@ElvUI_Libraries/Core/LibRangeCheck-3.0-26/LibRangeCheck-3.0.lua"]:837: in function <...braries/Core/LibRangeCheck-3.0/LibRangeCheck-3.0.lua:832>
[string "=(tail call)"]: ?
[string "@ElvUI_Libraries/Core/LibRangeCheck-3.0-26/LibRangeCheck-3.0.lua"]:912: in function <...braries/Core/LibRangeCheck-3.0/LibRangeCheck-3.0.lua:893>
[string "=(tail call)"]: ?
[string "@ElvUI/Core/Modules/UnitFrames/Elements/Range.lua"]:14: in function <...Ons/ElvUI/Core/Modules/UnitFrames/Elements/Range.lua:13>
[string "=(tail call)"]: ?
[string "@ElvUI/Core/Modules/UnitFrames/Elements/Range.lua"]:47: in function `UpdateRange'
[string "@ElvUI_Libraries/Core/oUF_Plugins/oUF_Fader.lua"]:99: in function <...ddOns/ElvUI_Libraries/Core/oUF_Plugins/oUF_Fader.lua:70>
[string "=(tail call)"]: ?
[string "@ElvUI_Libraries/Core/oUF_Plugins/oUF_Fader.lua"]:150: in function <...ddOns/ElvUI_Libraries/Core/oUF_Plugins/oUF_Fader.lua:144>

Locals:
_ = Frame {
}
event = "ADDON_ACTION_BLOCKED"
events = <table> {
}
```

**Comment by Skamer** [comment]
It's on Vanilla or Cataclym ?

**Comment by Zeffuro** [comment]
Oops forgot to mention that, it's Vanilla (Classic_Era)

**Comment by Skamer** [comment]
I have some questions:
- Do you encounter the problem directly after the addon loading or only when accepting/completing/doing quests ? 
- Do you do a  /reload fix the problem ?

**Comment by Skamer** [comment]
Can you do a test with only  Syling Tracker and Scorpio enabled (this is to be sure there are no incompatibility with an another addon). 

Have you changed the default background for each quest in Syling Tracker ? (i see the background is transparent). 

The fix i pushed in dev is for resolving an issue where the quests are sometimes not loaded on the first loading or when the cache folder is deleted.

**Comment by Zeffuro** [comment]
Yes I did change the default options (but for the next test all addons are disabled and I reset my SylingTracker options fully), I was just messing around to see if I can get the style closer to the style I have/had in Kaliel, I'm not quite there yet but I'm sure I'll be able to when you add some more features, it's mostly spacing I'm looking for and maybe a drop shadow option on the text.
![opera_2024-09-27_18-28-09](https://github.com/user-attachments/assets/6c1ed212-bd60-4da5-8530-6c4cf9d5111d)


I might have found the "issue"?
Apparently you can't "track" every single quest in Vanilla WoW?

Here is another video and near the end while I'm flying I'm playing with the shiftclicking to track option and I can get some quests to show but not all (after reloading).

https://puu.sh/KfKoD/a71426b019.mp4

I don't know if this is something that (previously Kaliel) and Questie circumvent it and just pull them all from the quest log or if they only used to show up in Kaliel because of Questie compatibility.

If there's a way to get all the quests to show up that'd be amazing.
